po/LINGUAS

```diff
@@ -5,6 +5,7 @@ da
 de
 el
 es
+et
```

Comment:
Changing LINGUAS here is not needed. Better to keep this in a separate commit with the et.po translation.

Reply:
OK, done.